File: test/fixtures/clojure/get_with_form.clj
(require '[clj-http.client :as client])

(client/post "http://localhost:28139/v3" {:basic-auth ["test" ""]
                                          :multipart [{:name "from" :content "test@tester.com"}
                                                      {:name "to" :content "devs@tester.net"}
                                                      {:name "subject" :content "Hello"}
                                                      {:name "text" :content "Testing the converter!"}]})

File: test/fixtures/kotlin/get_with_form.kt
import java.io.IOException
import okhttp3.Credentials
import okhttp3.MultipartBody
import okhttp3.OkHttpClient
import okhttp3.Request

val client = OkHttpClient()

val credential = Credentials.basic("test", "");

val requestBody = MultipartBody.Builder()
  .setType(MultipartBody.FORM)
  .addFormDataPart("from", "test@tester.com")
  .addFormDataPart("to", "devs@tester.net")
  .addFormDataPart("subject", "Hello")
  .addFormDataPart("text", "Testing the converter!")
  .build()

val request = Request.Builder()
  .url("http://localhost:28139/v3")
  .post(requestBody)
  .header("Authorization", credential)
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}
